slack_sdk/models/blocks/blocks.py

```diff
@@ -307,6 +359,23 @@ def __init__(
         """A block that collects information from users - it can hold a plain-text input element,
         a select menu element, a multi-select menu element, or a datepicker.
         https://api.slack.com/reference/block-kit/blocks#input
+
+        Args:
+            label (required): A label that appears above an input element in the form of a text object
+                that must have type of plain_text. Maximum length for the text in this field is 2000 characters.
+            element (required): An plain-text input element, a checkbox element, a radio button element,
+                a select menu element, a multi-select menu element, or a datepicker.
+            block_id: A string acting as a unique identifier for a block. If not specified, one will be generated.
+                Maximum length for this field is 255 characters.
+                block_id should be unique for each message or view and each iteration of a message or view.
+                If a message or view is updated, use a new block_id.
+            hint: An optional hint that appears below an input element in a lighter grey.
+                It must be a a text object with a type of plain_text.
```

Comment:
```suggestion
                It must be a text object with a type of plain_text.
```

Reply:
Thanks. Perhaps, I need to send a patch to the original document as well.

Reply:
@seratch Ah, I didn't realize you borrowed it directly from api.slack.com. I can send a patch for the original documentation and add you as a reviewer 👍🏻

Reply:
Fixed `a a text object` on https://api.slack.com/reference/block-kit/blocks#input_fields

Reply:
@mwbrooks You are so quick 🚀  Thank you!